tasks/sanity-label-check.yaml

```diff
@@ -19,7 +19,7 @@ spec:
   steps:
     - name: basic-sanity-checks-required-labels
       image: quay.io/redhat-appstudio/hacbs-test:stable
-      workingDir: $(workspaces.workspace.path)/hacbs/$(context.task.name)
+      workingDir: $(workspaces.workspace.path)/hacbs/$(context.task.name)-$(inputs.params.POLICY_NAMESPACE)
```

Comment:
what is `inputs.params`? Cannot see it in https://tekton.dev/docs/pipelines/variables/#variables-available-in-a-task

Reply:
Weird, getting the parameter seems to work both with `params.POLICY_NAMESPACE` and `inputs.params.POLICY_NAMESPACE`.
Switched to `params` to be in line with the docs.

Reply:
Just figured it out, the inputs.params was used in v1alpha1 https://tekton.dev/docs/pipelines/migrating-v1alpha1-to-v1beta1/#changes-to-input-parameters

Reply:
I see that it's used multiple times already, changed from Alpha to beta -> https://tekton.dev/docs/pipelines/migrating-v1alpha1-to-v1beta1/#changes-to-fields , so we should probably update other occurrence in separate PR.

Reply:
Agreed, will raise it after this one.